### ISSUE TITLE
question: how can i download elevenlabs response locally?

### ISSUE THREAD

**Issue by AbubakrChan** [elevenlabs/elevenlabs-python#87]
My question is how can i download elevenlabs response locally?

Thank you!!

**Comment by AbubakrChan** [comment]
> https://github.com/elevenlabs/elevenlabs-python/blob/main/API.md#save
> 
> ```python
> from elevenlabs import generate, save
> 
> audio = generate(
>     text="Hi there",
>     voice="Arnold",
>     model='eleven_multilingual_v1'
> )
> 
> save(audio, 'myvoice.mp3')
> ```

Hey Thank you soo much for your prompt response!!
I will try this out right now.

**Comment by AbubakrChan** [comment]
I am building a chatbot in gradio using whisper and elevenlabs, so that user can talk to and listen to the chatbot:

![image](https://github.com/elevenlabs/elevenlabs-python/assets/89600478/9a0435a9-7b48-4a25-98fb-be55c6ebbb7e)

When running it locally I was using elevenlabs Play( ) function to play the audio response and everything was working fine.

And now when I have deployed it on heroku everything is working same as it is except that the elevenlabs audio is not played. 

I noticed that previously the audio was played on my system and not on the website (when running locally), that's why when i even close the website the audio was still playing and now when it is deployed on internet it is no longer working.

Searching for solution of this I created this issue so that I can save the audio in some database and later play it.....
But this may not work either because my code is on huggingface and the saved file wouldn't be saved there in the repo...

I highly require your help @flavioschneider in this regard, this issue is soo concerning to me.

I truly appreciate your efforts!!

Thank you for your help.
